logic.py

```diff
@@ -21,14 +23,32 @@ def tuple2list(ExTuple):
     return ExList 
 
 def SQLQuery(statment):
-    cursor.execute(statment)
-    conn.commit()
-    data = logic.tuple2list(cursor.fetchall()) 
-    return data
+    try:
+        cursor.execute(statment)
+        conn.commit()
+        data = logic.tuple2list(cursor.fetchall())
+        status = "OKAY"
+        purpose = "PRODUCTION"
+        SQL_log(statment,status,purpose)
+        return data
+    except:
+        status = "ERROR"
+        purpose = "PRODUCTION"
+        SQL_log(statment,status,purpose)
+    
 
 def SQLInsertQuery(statment):
-    cursor.execute(statment)
-    conn.commit()
+    try:
```

Comment:
We shall try to remove repeated code in both the methods.

Reply:
the repeated code we are seeing, isn't repeated code, the variables are being set to different values